Block/Cart/Eligibility.php

```diff
@@ -68,18 +68,16 @@ public function __construct(
         $this->logger = $logger;
         $this->config = $config;
         $this->eligibilityHelper = $eligibilityHelper;
-        $this->logger->info('Construct Cart Eligibility',[]);
         $this->checkEligibility();
         $this->availabilityHelper = $availabilityHelper;
     }
 
-    public function checkEligibility()
+    public function checkEligibility():void
     {
         try {
             $this->eligibilityHelper->checkEligibility();
         } catch (\InvalidArgumentException $e) {
```

Comment:
3 more Exception are thrown by `            $this->eligibilityHelper->checkEligibility();`